amazon/ion/writer_text.py

```diff
@@ -341,7 +343,7 @@ def serialize(ion_event):
 _CONTAINER_DELIMITER_MAP = {
     IonType.STRUCT: b',',
     IonType.LIST: b',',
-    IonType.SEXP: b' ',
+    IonType.SEXP: {'normal': b' ', 'pretty': b''},
```

Comment:
Maybe it's personal preference, but I'd rather keep the delimiter maps as homogenous `IonType` -> `bytes` `dict`s. 

```python
_CONTAINER_DELIMITER_MAP_NORMAL = {
    IonType.STRUCT: b',',
    IonType.LIST: b',',
    IonType.SEXP: b' ',
}
_CONTAINER_DELIMITER_MAP_PRETTY = {
    IonType.STRUCT: b',',
    IonType.LIST: b',',
    IonType.SEXP: b'',
}
```

Then you'd make a `_serialize_container_delimiter_normal` and `serialize_container_delimiter_pretty` and leave `serialize` as it was. 

```python
_serialize_container_delimiter_normal = _serialize_container_factory('delimiter_normal', _CONTAINER_DELIMITER_MAP_NORMAL)
_serialize_container_delimiter_pretty = _serialize_container_factory('delimiter_pretty', _CONTAINER_DELIMITER_MAP_PRETTY)
```

Then in `_raw_writer_coroutine` you'd have something like
```python
serialize_container_delimiter = _serialize_container_delimiter_pretty if pretty else _serialize_container_delimiter_normal
delimiter = serialize_container_delimiter(container_event)
```

Reply:
That works with me, changed in next revision.